config/routes.rb

```diff
@@ -10,4 +10,5 @@
 
   resource :dashboard, only: :show
   resources :pairings, only: :create
+  post '/dashboard/send_flow' => "dashboards#send_flow"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.